### ISSUE TITLE
Suggestion. Going up and down keys

### ISSUE THREAD

**Issue by Sarowx** [radimitrov/CSharpShellApp#81]
Hello, I like ← → quick symbols for going left and right. Could you please add  ↑  ↓ symbols as well for going up and down? I tried to add it manually and it just pastes those symbols to the editor.

**Comment by radimitrov** [comment]
Yes. I'm going to rework some UI elements of the app and that will be in the defaults.
